src/pages/_app.tsx

```diff
@@ -111,10 +112,12 @@ function MyApp({ Component, pageProps }: AppProps): JSX.Element {
                           <DndProvider backend={HTML5Backend}>
                             <Hydrate state={dehydratedState}>
                               <CssBaseline />
-                              {getLayout(
-                                <Component {...restProps} />,
-                                restProps
-                              )}
+                              <NoSsr>
+                                {getLayout(
+                                  <Component {...restProps} />,
+                                  restProps
+                                )}
+                              </NoSsr>
```

Comment:
I read a conversation you had with @richardolsson . I read what `<NoSsr>` does. I guess I understand the concept of it. Testing works fine anyway!